### ISSUE TITLE
mfp_client_protection in catalystcenter_wireless_enterprise_ssid has incorrect schema

### ISSUE THREAD

**Issue by jolstorn** [CiscoDevNet/terraform-provider-catalystcenter#74]
In catalystcenter_wireless_enterprise_ssid, the provider expects "voicedata" or "data" as value for mfp_client_protection. 
Catalystcenter expects "Optional", "Disabled" or "Required".

```
│ Error: Invalid Attribute Value Match
│ 
│   with catalystcenter_wireless_enterprise_ssid.ssid["Guest"],
│   on wireless_ssid.tf line 29, in resource "catalystcenter_wireless_enterprise_ssid" "ssid":
│   29:     mfp_client_protection                 = each.value.mfp_client_protection
│ 
│ Attribute mfp_client_protection value must be one of: ["voicedata" "data"], got: "Optional"
```

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-catalystcenter/commit/301aad17f01fbb9948615db31314f13440a21aff

**Comment by danischm** [comment]
Fixed in v0.1.9 release.
